packages/botonic-plugin-contentful/src/contentful/manage/manage-asset.ts

```diff
@@ -51,6 +51,51 @@ export class ManageContentfulAsset {
     await this.writeAsset(context, oldAsset)
   }
 
+  async removeAsset(context: ManageContext, assetId: AssetId): Promise<void> {
+    const environment = await this.environment
+    try {
+      const asset = await environment.getAsset(assetId.id)
+      if (asset.isPublished()) await asset.unpublish()
+      await asset.delete()
+    } catch (e) {
+      throw new CmsException('ERROR while deleting asset', e)
+    }
+  }
+
+  async createAsset(
+    context: ManageContext,
+    title: string,
+    fileName: string,
+    contentType: string,
+    file: any,
+    description?: string
```

Comment:
having lots of parameters in a function can be a sign of having something hard to extend and maintain. I would do the following here:
* create an interface representing the asset you want to add
```
inferface NewAsset(AssetToCreate, ...) {
title: string,
fileName: string,
contentType: string,
file: any,
description?: string
}
```
* calling the function `createAsset` in this way `createAsset(context, {title: 'newTitle', fileName: 'fileName', ...}`

what do you think? :)